src/App.tsx

```diff
@@ -10,33 +12,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const handleTabSelect = (tab: Tab) => {
+    setSelectedTabId(tab.id);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${tabs.find(tab => selectedTabId === tab.id)?.title}`}
```

Comment:
it would be better to create a separate variable `activeTab` and use it here
```suggestion
        {`Selected tab is ${activeTab.title}`}
```